frontend_tests/node_tests/search_suggestion.js

```diff
@@ -788,13 +788,20 @@ run_test('stream_completion', () => {
     ];
     assert.deepEqual(suggestions.strings, expected);
 
+    query = '-stream:of';
+    suggestions = search.get_suggestions(query);
+    expected = [
+        "-stream:of",
+        "-stream:office",
+    ];
+    assert.deepEqual(suggestions.strings, expected);
+
     query = 'hel';
     suggestions = search.get_suggestions(query);
     expected = [
         "hel",
         "stream:dev+help",
     ];
-    assert.deepEqual(suggestions.strings, expected);
```

Comment:
Looks like this was accidentally removed.

Reply:
@timabbott updated, thanks for the review.